### ISSUE TITLE
Levels for talks and seminars

### ISSUE THREAD

**Issue by roed314** [roed314/seminars#298]
We're starting to get talks at various different levels (research seminars, grad student learning seminars, undergrad seminars, etc).  It would be good for organizers to be able to set these levels and then have filters on the browse page.

**Comment by AndrewVSutherland** [comment]
I think we should have a dropdown list for "type of seminar series" that would include colloquium, research seminar, learning seminar, undergraduate seminar, etc. and use this to replace the short description which is sometimes used for this and other times used to repeat the seminar name (and other times used for something completely different), leading to a rather jumbled looking seminar search page:

https://mathseminars.org/search

The fact that the short description field is used very inconsistently has been noted on the feedback page.  Having a free form field where it is unclear to users exactly what should go in the field is a recipe for inconsistency.  In the handful of cases where the short description actually contains information that is neither evident from the seminar name nor relevant to the level, this can be moved to the comments.

I'm not sure it is really necessary to be able to set this for individual talks  (but we certainly do want to be able to filter on it so it should be a column in the talks table), although I guess we could just copy it from the series and let organizers override it in the rare cases where it might make sense.

In any case, we should decide what we want to do about this soon -- the more series that are added the more of a headache it will be to change.

**Comment by edgarcosta** [comment]
I think @AndrewVSutherland is on the right track.

**Comment by AndrewVSutherland** [comment]
Another advantage of doing this is that it would free up some space on the seminar search page that would allow us to show other information, such as whether it is a conference or seminar series and its periodicity (weekly, bi-weekly, monthly, ....).  In some cases the periodicity is included in the short description, but usually not.

**Comment by roed314** [comment]
Here are the levels we came up with: research seminar, colloquium, learning seminar, advanced learning seminar, undergraduate seminar, general public.  Chime in if you can think of others.

**Comment by AndrewVSutherland** [comment]
I'm going to go ahead and implement this now, both because it's easy and because I am currently going through all the seminars to sanity check the access control settings -- I can check the level/type at the same time (if we make the default research seminar, there won't be many exceptions, but someone needs to actually look at each seminar to determine this.

**Comment by AndrewVSutherland** [comment]
This has been implemented and is now on master.  Currently it is only visible in the edit series and talk pages but once we are happy with it we should add it to the view talk and view seminar pages and consider replacing the "Description" column in browse series with level.

While writing the knowl I realized that "audience" might be a better term than "level" and wrote the knowl (and the knowl title) this way (but the database column is level -- we could painlessly change this to audience now if we wanted to).
